VirtusLab-Open-Source/strapi-plugin-comments
# [CU-20rfmhn] Settings page

## Ticket

[CU-20rfmhn](https://app.clickup.com/t/20rfmhn)

## Summary

What does this PR do/solve? 

Introduction of Settings page to dynamically configure the plugin.

## Test Plan

1. Run the app
2. Go to `Settings -> Comments plugin -> Configuration`
3. Play with config and see that it is reflected in the Publick REST & GraphQL API calls
